src/main/java/com/ioet/bpm/people/services/PasswordManagementService.java

```diff
@@ -0,0 +1,63 @@
+package com.ioet.bpm.people.services;
+
+import org.springframework.stereotype.Component;
+
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Base64;
+
+@Component
+public class PasswordManagementService {
```

Comment:
I'd encourage you to catch all the exceptions here instead. The thing is all the methods that you are using are throwing exceptions because the people who created the language wanted to make sure the developer take care of algorithm names and stuff like that so I think is safe to catch the errors and don't re-throw errors. That would be a good idea if you are retrieving the algorithm as a parameter for example.